scripts/minIO_import.py

```diff
@@ -0,0 +1,132 @@
+import concurrent.futures
+import os
+from pathlib import Path
+
+import boto3
+from tqdm import tqdm
+
+"""
+This script uploads genome files from collections NCBI source directory to the specified S3 bucket.
+
+Before running this script, make sure the following steps are completed:
+1. Start the SSH tunnel to the minIO server at NERSC:
+    ssh -L 9002:ci07:9002 $USERNAME@login1.berkeley.kbase.us -N
+2. Get the secret key from the minIO server in Rancher and set the environment variable:
+    i. Go to the minIO server in Rancher
+        Stacks → CI → cdm-minio -> ci-core-cdm-minio-1 → Execute Shell → echo $MINIO_ROOT_PASSWORD
```

Comment:
You can also get it from the Command tab without having to execute a shell

Reply:
👍 Didn't know it before.

Reply:
:+1: